### PR TITLE
W-022118 TDTM exclude username case bug

### PR DESCRIPTION
**Pull request by Julian88Tex**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by Julian88Tex** [comment]
Created this pull request since  https://github.com/SalesforceFoundation/Cumulus/pull/4527 had additional file chages that I had trouble removing from PR.

**Comment by Julian88Tex** [comment]
@jesslopez-sf if you have a second feel free to give it a test. After pulling the branch you can run this command:

`cci task run robot --org dev -o suites robot/Cumulus/tests/draft/tdtm_username_exclude.robot`

File: robot/Cumulus/tests/draft/tdtm_username_exclude.robot [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Update a Trigger Handler to Exclude a Username
+    [tags]  unstable
+
+    # Create a Contact so Trigger Handler records are created
+    &{account} =                 API Create Organization Account
```
Comment (Julian88Tex) [inline robot/Cumulus/tests/draft/tdtm_username_exclude.robot]: @jesslopez-sf added your suggestion here

**Comment by Safiyab** [comment]
@Julian88Tex The script looks great. I see that you added the "unstable" tag to your script. The **"Feature Test - Robot Unstable"** job only runs on branches that are named with the prefix **"feature/robot"**. That's why your last PR triggered the unstable job but this one did not.
Can you rename your branch with that prefix and force push the changes to remote ? Because we can't merge a script to master unless it passes in MetaCI build.
(Sorry for all the confusion here. We can sync up via a call if you have any questions related to this)

**Comment by Julian88Tex** [comment]
@Safiyab Ah thanks for info and all the help! I've created a new branch/PR... hopefully my last ;-) https://github.com/SalesforceFoundation/Cumulus/pull/4549

**Pull request closed without merging**